CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Projectiles/Boomerang/BoomerangIdle.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+
+namespace CrossPlatformDesktopProject.NPC
+{
+    class BoomerangIdle : INpcState
```

Comment:
Instead of having a completely empty class for the idle version of the Boomerang, you could try doing something similar to what we did with the equipables to make them disappear and no longer function when needed (using the TerminateBoomerang/TerminateBomb/etc. methods). It would make it cleaner than having an empty implementation of the interface for when you need this to happen, even though this solution does work for now.

Reply:
I'm not sure if I wanted to create a new class and move everything around for one method, though I might've been able to find an easier way to do it. But for now I added another boolean parameter that basically says to either draw/update it or not. Lmk how you feel about it and I can change it if need be.